Content.Server/Atmos/EntitySystems/AtmosphereSystem.HighPressureDelta.cs

```diff
@@ -201,50 +198,40 @@ private void ConsiderPressureDifference(GridAtmosphereComponent gridAtmosphere,
             if (!Resolve(uid, ref xform))
                 return;
 
-            // TODO ATMOS stuns?
-
-            var maxForce = MathF.Sqrt(pressureDifference) * 2.25f;
-            var moveProb = 100f;
-
-            if (component.PressureResistance > 0)
-                moveProb = MathF.Abs((pressureDifference / component.PressureResistance * MovedByPressureComponent.ProbabilityBasePercent) -
-                                     MovedByPressureComponent.ProbabilityOffset);
-
-            // Can we yeet the thing (due to probability, strength, etc.)
-            if (moveProb > MovedByPressureComponent.ProbabilityOffset && _robustRandom.Prob(MathF.Min(moveProb / 100f, 1f))
-                                                                      && !float.IsPositiveInfinity(component.MoveResist)
-                                                                      && (physics.BodyType != BodyType.Static
-                                                                          && (maxForce >= (component.MoveResist * MovedByPressureComponent.MoveForcePushRatio)))
-                || (physics.BodyType == BodyType.Static && (maxForce >= (component.MoveResist * MovedByPressureComponent.MoveForceForcePushRatio))))
+            // EXPLANATION:
+            // pressureDifference = Force of Air Flow on a given tile
+            // physics.Mass = Mass of the object potentially being thrown
+            // physics.InvMass = 1 divided by said Mass. More CPU efficient way to do division.
+            //
+            // Objects can only be thrown if the force of air flow is greater than the SQUARE of their mass or {SpaceWindMinimumMassThreshold}, whichever is heavier
+            // This means that the heavier an object is, the exponentially more force is required to move it
+            // The force of a throw is equal to the force of air pressure, divided by an object's mass. So not only are heavier objects
+            // less likely to be thrown, they are also harder to throw,
+            // while lighter objects are yeeted easily, and from great distance.
+            //
+            // For a human sized entity with a standard weight of 80kg and a spacing between a hard vacuum and a room pressurized at 101kpa,
+            // The human shall only be moved if he is either very close to the hole, or is standing in a region of high airflow
```

Comment:
```suggestion
            //INFO: EXPLANATION:
            // pressureDifference = Force of Air Flow on a given tile
            // physics.Mass = Mass of the object potentially being thrown
            // physics.InvMass = 1 divided by said Mass. More CPU efficient way to do division.
            //
            // Objects can only be thrown if the force of air flow is greater than the SQUARE of their mass or {SpaceWindMinimumMassThreshold}, whichever is heavier
            // This means that the heavier an object is, the exponentially more force is required to move it
            // The force of a throw is equal to the force of air pressure, divided by an object's mass. So not only are heavier objects
            // less likely to be thrown, they are also harder to throw,
            // while lighter objects are yeeted easily, and from great distance.
            //
            // For a human sized entity with a standard weight of 80kg and a spacing between a hard vacuum and a room pressurized at 101kpa,
            // The human shall only be moved if he is either very close to the hole, or is standing in a region of high airflow
```
Some of these should be XML docs for variables on the method itself and not just a comment in the code.
Almost *all* of the can just be in the summary or notes section of the XML docs.